main/webapp/modules/core/scripts/index/default-importing-sources/import-from-computer-form.html

```diff
@@ -1,5 +1,15 @@
 <form bind="form"><div class="grid-layout layout-normal"><table>
   <tr><td id="or-import-locate-files"></td></tr>
-  <tr><td><input type="file" multiple bind="fileInput" name="upload" /></td></tr>
+  <tr>
+    <td>
+
+      <div class="file-input-container">
+        <input type="file" multiple id="fileInput" name="upload" class="file-input" />
```

Comment:
I noticed that clicking the "Next »" button has no effect anymore as far as I can tell. This is likely related to your removal of the `bind` attribute on the `<input />`. Obviously it's quite critical to make sure this button is still functional :)